src/lib/y2storage/callbacks/probe.rb

```diff
@@ -55,18 +57,17 @@ def missing_command(message, what, command, used_features)
         # Redirect to error callback if no packages can be installed.
         return error(message, what) if used_features == 0 || Yast::Mode.installation
 
-        package_handler = Y2Storage::PackageHandler.new
-        package_handler.add_feature_packages(used_features)
+        packages = StorageFeaturesList.new(used_features).pkg_list
```

Comment:
This API looks confusing to me. Let's put aside that we are getting an integer from an OOP library to get the features that are used. Having a class called *BlaBlaList*, I would expect the constructor to receive an array (or a Set or "splat" args, etc.) of `StorageFeature` objects.

I had a look at the `StorageFeatureList` implementation and I would rather prefer an special constructor `from_bits` (or probably a better name).

However, it is just a matter of preference, and I can live with the current implementation.

Reply:
You are right, I will change it.

Reply:
Done in a later commit already added to this PR.